rapids-cmake/cpm/package_override.cmake

```diff
@@ -49,12 +49,15 @@ projects.
 
 .. note::
 
+  .. versionadded:: v23.10.00
```

Comment:
Do you want this to be inside the `.. note::`? It looks a little funny when rendered. Maybe put this above the note block.

Reply:
I moved the element to a separate note. I want to explicitly call this behavior change out so people understand what is happening so I am using a note element.